client/src/components/Upload.vue

```diff
@@ -272,7 +302,23 @@ function prepareFiles(files) {
                   :disabled="pendingUpload.uploading"
                 ></v-text-field>
               </v-col>
+              <v-col cols="1">
+                <v-list-item-action>
+                  <v-btn
+                    icon
+                    small
+                    @click="remove(pendingUpload)"
+                    :disabled="pendingUpload.uploading"
+                  >
+                    <v-icon>mdi-close</v-icon>
+                  </v-btn>
+                </v-list-item-action>
+              </v-col>
             </v-row>
+            <v-switch
+              label="Create Folder"
+              v-model="pendingUpload.createFolder"
+            />
```

Comment:
Sorry for the large diff here, but it's actually a small suggestion:

```suggestion
              <v-col cols="auto">
                <v-switch
                  label="Create Folder"
                  v-model="pendingUpload.createFolder"
                />
              </v-col>
              <v-col cols="1">
                <v-list-item-action>
                  <v-btn
                    class="mt-2"
                    icon
                    small
                    @click="remove(pendingUpload)"
                    :disabled="pendingUpload.uploading"
                  >
                    <v-icon>mdi-close</v-icon>
                  </v-btn>
                </v-list-item-action>
              </v-col>
            </v-row>
            <v-list-item-subtitle>
```

This just moves the close icon to be at the end of the list, moves the `v-switch` to be part of the `v-row`, and vertically centers the close icon, as it looked to be a bit high before. This is what it looks like:

![Screenshot from 2020-04-03 17-26-55](https://user-images.githubusercontent.com/11370025/78406361-75e7e300-75d0-11ea-9bf7-811423b154f1.png)

Feel free to push back, just a suggestion.

Reply:
I tried placing it a few different places to see what worked the best.  I tried before and after the text field.  Directly above or below Folder Name and others.  I don't know which is the best.  I also thought about changing it to a checkbox, just so the toggle doesn't take up as much horizontal space.

![image](https://user-images.githubusercontent.com/61746913/78500770-6d092580-7726-11ea-9770-986839b165ef.png)

Reply:
I like the checkbox, I still think having it on the right makes more sense, but it's your call.

Reply:
I think the hierarchy of decisions you make when uploading a file or files is first whether you want to create a folder for the files or not.  Toggling that checkbox causes the input field label to change and the visibility of the FPS input field to change as well.  It might be better to have changes occur after the toggle (left to right) instead of having them happen before the toggle.

Reply:
I think I agree with Bryon here.